css/about.css

```diff
@@ -0,0 +1,183 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
You can create a new CSS file and put the reset/normalize CSS in there and then insert it on each page before you insert index.css or about.css so you don't have to copy and paste it into every file.